index.js

```diff
@@ -43,8 +43,8 @@ async function main() {
 
         lines.push(
             [
-                name.substring(0,13).padEnd(15),
-                generateBarChart(plays, 31).substring(0, 27),
```

Comment:
btw why was a bar width 31 generating then truncated to 27? Shouldn't it have been width 27 in the first place?

Reply:
i believe it was modified because github's gist size was changed.